src/test/java/api/requests/ItemsInTransitReportTests.java

```diff
@@ -0,0 +1,190 @@
+package api.requests;
+
+import api.support.APITests;
+import api.support.builders.CheckInByBarcodeRequestBuilder;
+import api.support.builders.RequestBuilder;
+import api.support.http.InventoryItemResource;
+import api.support.http.ResourceClient;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.folio.circulation.domain.ItemStatus;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+public class ItemsInTransitReportTests extends APITests {
+
+  private static final String LOCATION_NAME = "name";
+  private static final String LOCATION_CODE = "code";
+  private static final String LIBRARY = "libraryName";
+  private static final String STATUS_KEY = "status";
+  private static final String BARCODE_KEY = "barcode";
+  private static final String TITLE = "title";
+  private static final String CONTRIBUTORS = "contributors";
+  private static final String DESTINATION_SERVICE_POINT = "inTransitDestinationServicePointId";
+
+  @Test
+  public void reportIsEmptyWhenThereAreNoItemsInTransit()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    assertTrue(items.isEmpty());
+  }
+
+  @Test
+  public void reportIncludeItemInTransit()
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource steve = usersFixture.steve();
+    final UUID secondServicePointId = servicePointsFixture.cd2().getId();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet);
+    createRequest(smallAngryPlanet, steve, secondServicePointId);
+    loansFixture.checkInByBarcode(smallAngryPlanet);
+
+    List<JsonObject> items = ResourceClient.forItemsInTransitReport(client).getAll();
+
+    verifyResponse(smallAngryPlanet, secondServicePointId, items);
+  }
+
+  private void createRequest(InventoryItemResource smallAngryPlanet, IndividualResource steve, UUID secondServicePointId)
+    throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+    RequestBuilder secondRequestBuilderOnItem = new RequestBuilder()
+      .open()
+      .hold()
+      .withPickupServicePointId(secondServicePointId)
+      .forItem(smallAngryPlanet)
+      .by(steve);
+    requestsClient.create(secondRequestBuilderOnItem);
+  }
```

Comment:
As this method is used by multiple tests, I think it would be better placed at the bottom of the class rather than interspersed with the tests

Reply:
Done